lib/CrashHandler/CrashHandler.cpp

```diff
@@ -407,14 +410,18 @@ void logBacktrace() try {
           // this is a stack frame we want to display
           memset(&buffer[0], 0, sizeof(buffer));
           char* p = &buffer[0];
-          appendNullTerminatedString("frame ", p);
-          if (frame < 10) {
+          appendNullTerminatedString("frame ", BYTES_LEFT(buffer, p), p);
+          if (frame < 10 && BYTES_LEFT(buffer, p) > 0) {
             // pad frame id to 2 digits length
             *p++ = ' ';
           }
-          p += arangodb::basics::StringUtils::itoa(uint64_t(frame), p);
+          if (BYTES_LEFT(buffer, p) > 19) {
+            p += arangodb::basics::StringUtils::itoa(uint64_t(frame), p);
+          }
 
-          appendAddress(pc, base, p);
+          if (BYTES_LEFT(buffer, p) > 16) {
```

Comment:
IMHO `appendAddress()` can append up to 23 bytes.

Reply:
Fixed in the meantime.